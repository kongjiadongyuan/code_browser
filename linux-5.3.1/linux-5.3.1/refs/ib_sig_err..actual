<dec f='linux-5.3.1/include/rdma/signature.h' l='117' type='u32'/>
<offset>64</offset>
