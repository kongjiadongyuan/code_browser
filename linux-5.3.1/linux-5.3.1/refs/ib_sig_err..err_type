<dec f='linux-5.3.1/include/rdma/signature.h' l='115' type='enum ib_sig_err_type'/>
<offset>0</offset>
