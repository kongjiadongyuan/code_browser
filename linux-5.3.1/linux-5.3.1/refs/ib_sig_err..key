<dec f='linux-5.3.1/include/rdma/signature.h' l='119' type='u32'/>
<offset>192</offset>
