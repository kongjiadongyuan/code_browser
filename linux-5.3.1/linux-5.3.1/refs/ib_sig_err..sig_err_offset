<dec f='linux-5.3.1/include/rdma/signature.h' l='118' type='u64'/>
<offset>128</offset>
