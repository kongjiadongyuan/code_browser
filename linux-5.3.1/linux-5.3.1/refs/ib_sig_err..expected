<dec f='linux-5.3.1/include/rdma/signature.h' l='116' type='u32'/>
<offset>32</offset>
